packages/i18n/tests/index.spec.ts

```diff
@@ -4,58 +4,59 @@ import { required, between } from '@/rules';
 import { localize, setLocale } from '@/i18n';
 import { mountWithHoc, setValue, flushPromises } from '../../vee-validate/tests/helpers';
 
-defineRule('required', required);
```

Comment:
Prefixed the names of them all (why I did a `describe` in the first place) and put them in a section at the bottom, and removed the `describe` and it looks a lot cleaner 🧼 .

The intent was to avoid the test name change (since I copied many tests, and simply added the prefix/suffix options) and group them - but yeah that diff was messy 😅